crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -926,6 +927,33 @@ impl SweetConductor {
             Ok(all_integrated)
         })
     }
+
+    /// Check if all ops of a specific author have been integrated in the DHT database.
+    pub fn all_ops_of_author_integrated(
```

Comment:
The other method above checks whether ops of all authors are integrated. That doesn't help in tests where an agent is discovered that hasn't declared a full storage arc, from whom an agent could fetch. So this method just checks all ops of a particular author.